frontend/src/components/Ingredients/Ingredients.tsx

```diff
@@ -1,19 +1,91 @@
-import { useEffect, useState } from 'react'
-import { fetchAllIngredients } from '../apiCalls'
-import './Ingredients.css'
+import { Ingredient } from '../apiCalls';
 
+interface Props {
+  ingredients: Array<Ingredient>;
+  setIngredients: Function;
```

Comment:
We can type this more specifically as a function that accepts an array of ingredients as params and returns nothing (`void`):
```suggestion
  setIngredients: (ingredients: Array<Ingredient>) => void;
```